src/NetAnalyzers/Core/AnalyzerReleases.Unshipped.md

```diff
@@ -7,6 +7,7 @@ CA1021 | Design | Disabled | AvoidOutParameters, [Documentation](https://docs.mi
 CA1045 | Design | Disabled | DoNotPassTypesByReference, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca1045)
 CA1047 | Design | Info | DoNotDeclareProtectedMembersInSealedTypes, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca1047)
 CA1069 | Design | Info | EnumShouldNotHaveDuplicatedValues, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca1069)
+CA1070 | Design | Info | DoNotDeclareEventFieldsAsVirtual, [Documentation](https://docs.microsoft.com/visualstudio/code-quality/ca1070)
```

Comment:
Please file a doc issue to add documentation for this rule.

Reply:
https://github.com/MicrosoftDocs/visualstudio-docs/issues/5151